### PR TITLE
ECIP-1000: Updating process.png file

### PR DESCRIPTION
**Pull request by TokenHash**
The current process.png is outdated and wrong with regards to arrows. It is creating confusion about direction of status adjustments.

### CODE REVIEW

File: _specs/ecip-1102.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+ecip: 1102
```
Comment (q9f) [inline _specs/ecip-1102.md:2]: something went wrong here, donald

**Comment by gitr0n1n** [comment]
@TokenHash It looks like you accidentially included your ECIP 1102 file in this proposal. Just delete that document from this PR and leave the updated diagram and we should be able to review it and approve. Thanks!

File: _specs/ecip-1102.md (hunk 1 above)
Comment (TokenHash) [reply _specs/ecip-1102.md]: i think i solved it

**Comment by TokenHash** [comment]
@gitr0n1n I think I did what you said. Perhaps now is ready to update the image.

**Review by developerkevin — APPROVED** [review]
LGTM! All necessary changes have been completed.

**Pull request merged**